Updated the project to use Qiq3.

suzumaze opened the pull request:
Due to the breaking changes between Qiq1 and Qiq2, the versions were separated, and the necessary adjustments were made to support Qiq3.
The auto-loading of helpers, which was previously implemented in the HelperLocator, has not been re-implemented in this update.
Added an example of registering custom helpers in the demo project.

Qiq3を使うように変更しました。
Qiq1から2で大きなブレイクがあったため、バージョンを分ける必要がありそうです。
また、ヘルパーロケーターに実装されていたオートロードは実装しませんでした。
そのかわりに、デモでカスタムヘルパーの登録例を追加しています。

suzumaze committed:
Update PHP and Qiq

suzumaze committed:
Update .gitignore

suzumaze committed:
Update dependencies to use Qiq3 instead of Qiq1

suzumaze committed:
Update demo and refactor custom helper binding

suzumaze committed:
Remove StubGen and other unused files

suzumaze committed:
Add .gitkeep

koriym commented:
@suzumaze テストを動くようにしましょうか

suzumaze committed:
Update the PHP version to 8.3 in the GitHub Workflow

koriym commented:
@suzumaze テストにPHP 8.1, 8.2も追加しましょうか

koriym commented on .github/workflows/continuous-integration.yml:
```diff
@@ -15,9 +15,6 @@ jobs:
-          - '8.0'
-          - '8.1'
-          - '8.2'
           - '8.3'
```
```suggestion
          - '8.1'
          - '8.2'
          - '8.3'
```

suzumaze replied:
こちら対応いたしました！

suzumaze committed:
Add PHP versions 8.1 and 8.2 to the test matrix

koriym commented on composer.json:
```diff
@@ -9,8 +9,8 @@
     "require": {
-        "php": "^8.0",
-        "qiq/qiq": "^1.0",
+        "php": "^8.3",
```
```suggestion
        "php": "^8.1",
```

koriym committed:
Update composer.json

koriym commented:
@coderabbitai full review

koriym commented:
@suzumaze PR先を2.xにお願いします！🙏

koriym commented:
@coderabbitai 日本語でお願いします。

koriym commented:
@coderabbitai 以前のコメントをもう一度日本語でお願いします。

koriym commented:
@jingu gentle reminder

koriym committed:
Upgrade to Psalm 6 and improve type safety

- Upgrade vimeo/psalm from ^4.2 to ^6.0
- Upgrade psalm/plugin-phpunit from ^0.13 to ^0.19
- Change QiqRenderer to use Template instead of Engine for proper __invoke() support
- Add final modifier to QiqErrorPage and QiqErrorPageRenderer
- Fix $headers type annotation in QiqErrorPage using psalm-import-type
- Remove redundant (int) cast in QiqErrorHandler
- Add _setBindings method to FakeWeavedRo for ray/aop compatibility
- Remove unused exception classes (LogicException, RuntimeException)
- Remove psalm-baseline.xml (no longer needed)
- Clean up phpstan.neon and psalm.xml suppressions

koriym committed:
Update GitHub Actions to use actions/checkout@v4 and actions/cache@v4

koriym committed:
Add CHANGELOG.md for 2.0.0 release
